pkg/chunked/internal/errors.go

```diff
@@ -0,0 +1,8 @@
+package internal
+
+import (
+	"errors"
+)
+
+// ErrPartialContentNotAvailable is returned when the requested content does not support partial content retrieval
+var ErrPartialContentNotAvailable = errors.New("partial content not available")
```

Comment:
This is correct as stated, but I think if the specific thing we want to control is _fallback_, the error name should say so. (Also the existence of `convert_images` complicates the discussion of “partial content”…)

`ErrFallbackToOrdinaryLayerDownload`?